enginerpc/engine_impl.cc

```diff
@@ -529,6 +540,9 @@ grpc::Status engine_impl::AddHostComment(grpc::ServerContext* context
   });
 
   std::future<int32_t> result = fn.get_future();
+  std::string json_str;
+  google::protobuf::util::MessageToJsonString(*request,  &json_str);
+  logger_info("AddHostComment", json_str);
```

Comment:
It is not a good idea to translate messages to json. This is too expensive.
You can make logs directly using the message and maybe without all the fields of it.